smtp/init.lua

```diff
@@ -152,6 +152,27 @@ local function addr_spec(mailbox)
     return mailbox:match('^.*[ \t]*<(.-)>[ \t]*$') or mailbox
 end
 
+-- According to RFC 822#3.3 subject must contain characters with codes 0-127:
+-- https://tools.ietf.org/html/rfc822
+-- For compatibility with legacy mail relays simply base64 encode subject field
+-- if char codes >127 is present
+local function encode_subject(subj)
+    local is_gt_127 = function(text)
```

Comment:
Do you really need to recreate this function for each call?

Reply:
Moved is_gt_127() out of encode_subject() scope

Reply:
I squashed this commit into the first one.